src/Octane/OctaneClient.php

```diff
@@ -20,29 +20,18 @@ class OctaneClient implements Client
 {
     /**
      * The Octane worker.
-     *
-     * @var \Laravel\Octane\Worker
      */
-    protected static $worker;
+    private Worker $worker;
```

Comment:
Have you tested these `static` to `$this` changes and whether it keeps the client alive between requests?

Reply:
You made me doubt so I double checked, yes the client and worker is kept alive (here across multiple requests):
<img width="1318" alt="image" src="https://user-images.githubusercontent.com/720328/220307245-40b6808c-6d02-48c5-a753-c6c1a193568e.png">

But yeah I removed the static variable but also all uses of this variable are now non-static (`$this->` instead of `self::`).

It basically goes like this: the `OctaneHandler` has 1 `OctaneClient` that has 1 `Octane\Worker`.

`OctaneHandler` is kept alive thanks to `BREF_LOOP_MAX`, so the rest is as well. The static variables are not really necessary.